Databases/CommandSchedulerDb/Scheduler/Tables/ScheduledCommand.sql

```diff
@@ -15,8 +15,8 @@
 
 
 GO
-CREATE CLUSTERED INDEX [IX_AppliedTime_Clock_Id_FinalAttemptTime_DueTime]
-    ON [Scheduler].[ScheduledCommand]([AppliedTime] ASC, [Clock_Id] ASC, [FinalAttemptTime] ASC, [DueTime] ASC);
+CREATE CLUSTERED INDEX [IX_AggregateId_DueTime_Clock_Id_AppliedTime_FinalAttemptTime]
+    ON [Scheduler].[ScheduledCommand]([AggregateId] ASC, [DueTime] ASC, [Clock_Id] ASC, [AppliedTime] ASC, [FinalAttemptTime] ASC);

```

Comment:
It is in the .cs file that creates the table 

Sent from my iPhone

> On Apr 23, 2016, at 10:18 AM, Jon Sequeira <notifications@github.com> wrote:
> 
> In Databases/CommandSchedulerDb/Scheduler/Tables/ScheduledCommand.sql:
> 
> > @@ -15,8 +15,8 @@
> >  
> >  
> >  GO
> > -CREATE CLUSTERED INDEX [IX_AppliedTime_Clock_Id_FinalAttemptTime_DueTime]
> > -    ON [Scheduler].[ScheduledCommand]([AppliedTime] ASC, [Clock_Id] ASC, [FinalAttemptTime] ASC, [DueTime] ASC);
> > +CREATE CLUSTERED INDEX [IX_AggregateId_DueTime_Clock_Id_AppliedTime_FinalAttemptTime]
> > +    ON [Scheduler].[ScheduledCommand]([AggregateId] ASC, [DueTime] ASC, [Clock_Id] ASC, [AppliedTime] ASC, [FinalAttemptTime] ASC);
> I'm not seeing it. Which file?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>